fashion-example-query/fashion/app.py

```diff
@@ -0,0 +1,180 @@
+    9: 'Ankle boot'
+}
+
+def get_mapped_label(label_int):
+    """
+    Get a label_int and return the description of that label
+    label_int	Description
+    0	        T-shirt/top
+    1	        Trouser
+    2	        Pullover
+    3	        Dress
+    4	        Coat
+    5	        Sandal
+    6	        Shirt
+    7	        Sneaker
+    8	        Bag
+    9	        Ankle boot
+    """
+    return label.get(label_int, "Invalid tag")
+
+
+def print_result(resp):
+    for d in resp.search.docs:
+        vi = d.uri
+        result_html.append(f'<tr><td><img src="{vi}"/></td><td>')
+        for kk in d.matches:
+            kmi = kk.uri
+            result_html.append(f'<img src="{kmi}" style="opacity:{kk.score.value}"/>')
+            # k['score']['explained'] = json.loads(kk.score.explained)
+        result_html.append('</td></tr>\n')
+
+
+def write_html(html_path):
+    with open(resource_filename('jina', '/'.join(('resources', 'helloworld.html'))), 'r') as fp, \
+            open(html_path, 'w') as fw:
+        t = fp.read()
+        t = t.replace('{% RESULT %}', '\n'.join(result_html))
+        fw.write(t)
+
+    url_html_path = 'file://' + os.path.abspath(html_path)
+
+    try:
+        webbrowser.open(url_html_path, new=2)
+    except:
+        pass
+    finally:
+        default_logger.success(f'You should see a "hello-world.html" opened in your browser, '
+                               f'if not you may open {url_html_path} manually')
+
+    colored_url = colored('https://opensource.jina.ai', color='cyan', attrs='underline')
+    default_logger.success(
+        f'🤩 Intrigued? Play with "jina hello-world --help" and learn more about Jina at {colored_url}')
+
+
+def load_mnist(path):
+    with gzip.open(path, 'rb') as fp:
+        return np.frombuffer(fp.read(), dtype=np.uint8, offset=16).reshape([-1, 784])
+
+
+def load_labels(path):
+    with gzip.open(path, 'rb') as fp:
+        labels = np.frombuffer(fp.read(), dtype=np.uint8, offset=16).reshape([-1, 1])
+        return labels
+
+
+def download_data(targets, download_proxy=None):
+    opener = urllib.request.build_opener()
+    if download_proxy:
+        proxy = urllib.request.ProxyHandler({'http': download_proxy, 'https': download_proxy})
+        opener.add_handler(proxy)
+    urllib.request.install_opener(opener)
+    with ProgressBar(task_name='download fashion-mnist', batch_unit='') as t:
+        for v in targets.values():
+            if not os.path.exists(v['filename']):
+                urllib.request.urlretrieve(v['url'], v['filename'], reporthook=lambda *x: t.update(1))
+            if v['filename'] == './workspace/labels-original':
+                v['data'] = load_labels(v['filename'])
+            else:
+                v['data'] = load_mnist(v['filename'])
+
+
+def index():
+    for j in range(num_docs):
+        d = jina_pb2.Document()
+        d.embedding.CopyFrom(array2pb(targets['index']['data'][j]))
+        label = (targets['labels']['data'][j]).item()
```

Comment:
Shouldn't it work like this?
```suggestion
        label = targets['labels']['data'][j][0]
```